crates/bevy_window/src/window.rs

```diff
@@ -160,6 +160,24 @@ pub struct Window {
     pub fit_canvas_to_parent: bool,
     /// Stores internal state that isn't directly accessible.
     pub internal: InternalWindowState,
+    /// Should the window use Input Method Editor?
+    ///
+    /// If enabled, the window will receive [`Ime`](crate::Ime) events instead of
+    /// [`ReceivedCharacter`](crate::ReceivedCharacter) or
+    /// [`KeyboardInput`](bevy_input::keyboard::KeyboardInput).
+    ///
+    /// IME should be enabled during text input, but not when you expect to get the exact key pressed.
+    ///
+    ///  ## Platform-specific
+    ///
+    /// - iOS / Android / Web: Unsupported.
+    pub ime_enabled: bool,
```

Comment:
Does it make sense to put these behind platform specific `#[cfg()]` if they are not supported for some platforms?

Reply:
No, that's not how this platforms specific things are handled here, see `transparent`, `decorations`, `always_on_top`, `canvas`, ...